tests/acceptance/file-details/api-download-test.js

```
@@ -0,0 +1,147 @@
+      return schema.dynamicscans.find(`${req.params.id}`)?.toJSON();
+    });
+
+    this.server.get('/v2/files/:id/dynamicscans', (schema, req) => {
+      const { limit, mode } = req.queryParams || {};
+
+      const results = schema.dynamicscans
+        .where({
+          file: req.params.id,
+          ...(mode ? { mode: Number(mode) } : {}),
+        })
+        .models.slice(0, limit ? Number(limit) : results.length);
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)